.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "assets/Font-Awesome"]
+	path = assets/Font-Awesome
+	url = https://github.com/FortAwesome/Font-Awesome.git
+[submodule "assets/bootstrap-sass"]
+	path = assets/bootstrap-sass
+	url = https://github.com/twbs/bootstrap-sass
```

Comment:
we can't as our build server environment is offline and I have no good place to prefetch the online stuff. So the submodule is the next best thing ;(